hash/spectrum_cass.xml

```diff
@@ -164540,9 +164868,22 @@ Tape 2 - Sida B: Star Wars
 	</software>
 
 	<software name="themagni">
-		<description>The Magnificent 7</description>
+		<description>The Magnificent Seven</description>
 		<year>1987</year>
 		<publisher>Ocean</publisher>
+		<info name="alt_title" value="The Magnificent 7"/>
+		<notes><![CDATA[
+Compilation contents:
+Tape 1 - Side A - Game 1: Wizball (Ocean)
+Tape 1 - Side A - Game 2: Head over Heels (Ocean)
+Tape 1 - Side B - Game 1: Arkanoid (Imagine Software)
+Tape 1 - Side B - Game 2: Cobra (Ocean)
+Tape 2 - Side A - Game 1: Frankie Goes to Hollywood (Ocean)
+Tape 2 - Side A - Game 2: The Great Escape (Ocean)
+Tape 2 - Side B - Game 1: Short Circuit (Ocean)
+Tape 2 - Side B - Game 2: Yie Ar Kung-Fu (Imagine Software)
+]]></notes>
```

Comment:
Validation insists that `<notes>` precede `<info>`.

Reply:
I already switch the order of the _tags_.